api/users/views.py

```diff
@@ -288,25 +287,23 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin
     view_category = 'users'
     view_name = 'user-nodes'
 
+    ordering = ('-date_modified')
+
     # overrides ODMFilterMixin
     def get_default_odm_query(self):
         user = self.get_user()
         return (
-            Q('contributors', 'eq', user) &
+            Q('contributors', 'contains', user._id) &
+            Q('is_folder', 'ne', True) &
             Q('is_collection', 'ne', True) &
-            Q('is_deleted', 'ne', True)
+            Q('is_deleted', 'ne', True) &
+            Q('is_registration', 'ne', True)
```

Comment:
The way I understand it, we're not claiming backwards compatibility yet - our API is public, but we haven't made a public announcement.  My changes aside, there is an error in the API.  We have separated the nodes and registrations into separate endpoints yet we are still documenting that you can filter on registrations in the nodes endpoint, when none will be returned.